2023-03-31 19:42:00 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:42:00 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4932.9MB, RSS 13.0MB     [log.py:88]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_31.txt     [logger.py:222]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:42:00 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:42:01 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:42:01 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-194201.csv'     [history.py:130]
2023-03-31 19:42:01 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:42:01 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:42:10 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:42:10 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:42:11 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:42:11 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:42:11 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:42:11 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:42:11 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:42:39 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:42:39 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5191.9MB, RSS 12.7MB     [log.py:88]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_32.txt     [logger.py:222]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:42:39 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:42:39 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-194201.csv'     [history.py:188]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-194239.csv'     [history.py:130]
2023-03-31 19:42:39 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:42:39 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:42:44 INFO     [MainThread  ] kivy.servicev: SHUTDOWN triggered     [serviceview.py:33]
2023-03-31 19:42:48 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 383, in idle     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     self.dispatch_input()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 334, in dispatch_input     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     post_dispatch_input(*pop(0))     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 302, in post_dispatch_input     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     wid.dispatch('on_touch_up', me)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/uix/behaviors/button.py", line 179, in on_touch_up     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     self.dispatch('on_release')     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_event.pyx", line 727, in kivy._event.EventDispatcher.dispatch     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_event.pyx", line 1307, in kivy._event.EventObservers.dispatch     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_event.pyx", line 1191, in kivy._event.EventObservers._dispatch     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/lang/builder.py", line 55, in custom_callback     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     exec(__kvlang__.co_value, idmap)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv", line 253, in <module>     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     root.ok_action()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/popups.py", line 27, in ok_action     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     self.ok_callback()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/serviceview.py", line 34, in shutdown     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     self.exec(['sudo', 'shutdown', 'now'])     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/serviceview.py", line 47, in exec     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     exit_code = call(command)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 351, in call     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     return p.wait(timeout=timeout)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1189, in wait     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     return self._wait(timeout=timeout)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1917, in _wait     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     (pid, sts) = self._try_wait(0)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/subprocess.py", line 1875, in _try_wait     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     (pid, sts) = os.waitpid(self.pid, wait_flags)     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:42:48 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:43:43 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:43:43 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5075.5MB, RSS 12.9MB     [log.py:88]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_33.txt     [logger.py:222]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:43:43 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:43:43 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:43:43 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:43:44 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 954, in run     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     self._run_prepare()     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 924, in _run_prepare     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     root = self.build()     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 80, in build     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     return DataLoggerWidget()     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 43, in __init__     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     super().__init__(**kwargs)     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/uix/boxlayout.py", line 145, in __init__     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     super(BoxLayout, self).__init__(**kwargs)     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/uix/layout.py", line 76, in __init__     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     super(Layout, self).__init__(**kwargs)     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/uix/widget.py", line 366, in __init__     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     self.apply_class_lang_rules(     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     Builder.apply(     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/lang/builder.py", line 540, in apply     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     self._apply_rule(     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/lang/builder.py", line 662, in _apply_rule     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     self._apply_rule(     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/lang/builder.py", line 620, in _apply_rule     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     cls = Factory_get(cname)     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/factory.py", line 147, in __getattr__     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr:     raise FactoryException('Unknown class <%s>' % name)     [logger.py:338]
2023-03-31 19:43:44 WARNING  [MainThread  ] kivy         : stderr: kivy.factory.FactoryException: Unknown class <ServiceScreen>     [logger.py:338]
2023-03-31 19:43:47 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:43:47 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:43:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:43:47 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:43:47 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:43:59 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:43:59 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4778.8MB, RSS 12.6MB     [log.py:88]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_34.txt     [logger.py:222]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:43:59 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:44:00 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:44:00 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-194400.csv'     [history.py:130]
2023-03-31 19:44:00 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:44:00 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:44:10 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:44:10 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:44:12 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:44:12 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:44:12 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:44:12 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:44:12 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:45:37 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:45:37 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4824.9MB, RSS 13.0MB     [log.py:88]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_35.txt     [logger.py:222]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:45:37 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:45:37 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-194537.csv'     [history.py:130]
2023-03-31 19:45:37 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:45:37 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:45:46 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:45:46 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:46:16 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:46:16 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5066.1MB, RSS 12.9MB     [log.py:88]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_36.txt     [logger.py:222]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:46:16 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:46:16 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-194537.csv'     [history.py:188]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-194616.csv'     [history.py:130]
2023-03-31 19:46:16 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:46:16 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:46:47 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:46:47 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:46:48 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:46:48 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:46:48 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:46:48 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:46:48 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:50:01 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:50:01 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5076.5MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_37.txt     [logger.py:222]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:50:01 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:50:01 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195001.csv'     [history.py:130]
2023-03-31 19:50:01 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:50:01 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:50:05 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 341, in mainloop     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:     self.window.mainloop()     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/core/window/window_sdl2.py", line 561, in mainloop     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr:     event = self._win.poll()     [logger.py:338]
2023-03-31 19:50:05 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:50:07 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:50:07 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:50:07 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:50:07 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:50:07 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:51:24 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:51:24 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5067.1MB, RSS 12.9MB     [log.py:88]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_38.txt     [logger.py:222]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:51:24 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:51:24 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195124.csv'     [history.py:130]
2023-03-31 19:51:24 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:51:24 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:52:24 INFO     [Thread-6    ] kivy.history : 'Closing CSV file '../logs/csv/signals-2023-03-31-195124.csv'     [history.py:138]
2023-03-31 19:52:24 INFO     [Thread-6    ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195224.csv'     [history.py:130]
2023-03-31 19:53:24 INFO     [Thread-6    ] kivy.history : 'Closing CSV file '../logs/csv/signals-2023-03-31-195224.csv'     [history.py:138]
2023-03-31 19:53:24 INFO     [Thread-6    ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195324.csv'     [history.py:130]
2023-03-31 19:54:24 INFO     [Thread-6    ] kivy.history : 'Closing CSV file '../logs/csv/signals-2023-03-31-195324.csv'     [history.py:138]
2023-03-31 19:54:24 INFO     [Thread-6    ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195424.csv'     [history.py:130]
2023-03-31 19:55:24 INFO     [Thread-6    ] kivy.history : 'Closing CSV file '../logs/csv/signals-2023-03-31-195424.csv'     [history.py:138]
2023-03-31 19:55:24 INFO     [Thread-6    ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195524.csv'     [history.py:130]
2023-03-31 19:56:24 INFO     [Thread-6    ] kivy.history : 'Closing CSV file '../logs/csv/signals-2023-03-31-195524.csv'     [history.py:138]
2023-03-31 19:56:24 INFO     [Thread-6    ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195624.csv'     [history.py:130]
2023-03-31 19:56:30 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:56:30 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:56:31 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:56:31 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:56:31 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:56:31 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:56:31 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:56:32 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:56:32 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5056.6MB, RSS 13.0MB     [log.py:88]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_39.txt     [logger.py:222]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:56:32 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:56:32 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195524.csv'     [history.py:188]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195624.csv'     [history.py:188]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195632.csv'     [history.py:130]
2023-03-31 19:56:32 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:56:32 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:56:40 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:56:40 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:56:41 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:56:41 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:56:41 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:56:41 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:56:41 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:56:56 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:56:56 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4798.7MB, RSS 12.6MB     [log.py:88]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_40.txt     [logger.py:222]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:56:56 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:56:56 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:56:56 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195524.csv'     [history.py:188]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195624.csv'     [history.py:188]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195632.csv'     [history.py:188]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195657.csv'     [history.py:130]
2023-03-31 19:56:57 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:56:57 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:56:59 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:56:59 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:57:00 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:57:00 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:57:00 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:57:00 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:57:00 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:57:49 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:57:49 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5086.0MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_41.txt     [logger.py:222]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:57:49 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:57:49 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195657.csv'     [history.py:188]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195749.csv'     [history.py:130]
2023-03-31 19:57:49 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:57:49 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:57:52 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:57:52 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:57:53 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:57:53 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:57:53 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:57:53 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:57:53 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:58:24 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:58:24 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4789.2MB, RSS 12.7MB     [log.py:88]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_42.txt     [logger.py:222]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:58:24 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:24 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195749.csv'     [history.py:188]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195824.csv'     [history.py:130]
2023-03-31 19:58:24 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:58:24 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:58:28 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:58:28 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:58:33 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:58:33 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:58:33 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:58:33 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:58:33 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:58:34 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:58:34 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4950.7MB, RSS 12.9MB     [log.py:88]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_43.txt     [logger.py:222]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:58:34 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:58:35 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:35 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195749.csv'     [history.py:188]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195824.csv'     [history.py:188]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:130]
2023-03-31 19:58:35 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:58:35 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:58:42 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:58:42 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:58:56 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:58:56 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:58:56 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:58:56 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:58:56 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:58:57 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:58:57 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4932.9MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_44.txt     [logger.py:222]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:58:57 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:57 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195824.csv'     [history.py:188]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:188]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:130]
2023-03-31 19:58:57 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:58:57 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:59:10 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:59:10 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:59:21 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:59:21 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:21 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:59:21 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:59:21 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:59:22 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:59:22 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4950.7MB, RSS 13.0MB     [log.py:88]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_45.txt     [logger.py:222]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:59:22 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:22 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:59:22 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195824.csv'     [history.py:188]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:188]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:188]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195923.csv'     [history.py:130]
2023-03-31 19:59:23 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:59:23 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:59:26 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:59:26 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:59:27 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:59:27 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:27 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:59:27 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:59:27 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:59:32 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:59:32 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 5057.7MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_46.txt     [logger.py:222]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:59:32 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:32 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195824.csv'     [history.py:188]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:188]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:188]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195923.csv'     [history.py:188]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195932.csv'     [history.py:130]
2023-03-31 19:59:32 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:59:32 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:59:35 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:59:35 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:59:36 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:59:36 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:36 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:59:36 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:59:36 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:59:40 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:59:40 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4932.9MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_47.txt     [logger.py:222]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:59:40 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:40 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:188]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:188]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195923.csv'     [history.py:188]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195932.csv'     [history.py:188]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195940.csv'     [history.py:130]
2023-03-31 19:59:40 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:59:40 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 19:59:45 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 19:59:45 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 19:59:53 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 19:59:53 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4788.2MB, RSS 12.8MB     [log.py:88]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_48.txt     [logger.py:222]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 19:59:53 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:53 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 19:59:53 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195835.csv'     [history.py:188]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:188]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195923.csv'     [history.py:188]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195932.csv'     [history.py:188]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195940.csv'     [history.py:188]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-195954.csv'     [history.py:130]
2023-03-31 19:59:54 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 19:59:54 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 20:00:03 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 776, in post_idle     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     self._process_events()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_clock.pyx", line 620, in kivy._clock.CyClockBase._process_events     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_clock.pyx", line 653, in kivy._clock.CyClockBase._process_events     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_clock.pyx", line 649, in kivy._clock.CyClockBase._process_events     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "kivy/_clock.pyx", line 218, in kivy._clock.ClockEvent.tick     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy_garden/graph/__init__.py", line 1241, in draw     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     for x, y in self.iterate_points():     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy_garden/graph/__init__.py", line 1139, in iterate_points     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     yield x_px(x), y_px(y)     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy_garden/graph/__init__.py", line 1047, in <lambda>     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     return lambda x: (funcx(x) - xmin) * ratiox + size[0]     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 20:00:03 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
2023-03-31 20:00:14 INFO     [MainThread  ] log          : 


--------------------------------------------------------------------
--- DATALOGGER APPLICATION RESTART  --------------------------------
--------------------------------------------------------------------

     [log.py:57]
2023-03-31 20:00:14 DEBUG    [Thread-1    ] root         : TICK - CPU temp unknown, memory usage VM 4924.5MB, RSS 12.8MB     [log.py:88]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Logger: Record log in /Users/austinibele/.kivy/logs/kivy_23-03-31_49.txt     [logger.py:222]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Kivy: v2.1.0     [__init__.py:509]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Kivy: Installed at "/usr/local/lib/python3.9/site-packages/kivy/__init__.py"     [__init__.py:510]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Python: v3.9.16 (main, Dec  7 2022, 10:15:13) 
[Clang 13.0.0 (clang-1300.0.29.30)]     [__init__.py:511]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Python: Interpreter at "/usr/local/opt/python@3.9/bin/python3.9"     [__init__.py:512]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Logger: Purge log fired. Processing...     [logger.py:170]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Logger: Purge finished!     [logger.py:187]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Factory: 189 symbols loaded     [factory.py:186]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Image: Providers: img_tex, img_imageio, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)     [__init__.py:145]
2023-03-31 20:00:14 ERROR    [MainThread  ] kivy.__main__: Starting application     [main.py:84]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy.mqttclie: NOT starting MQTT client because of config with empty broker     [mqttclient.py:61]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Text: Provider: sdl2     [__init__.py:67]
2023-03-31 20:00:14 WARNING  [MainThread  ] kivy         : Factory: Ignored class "Label" re-declaration. Current -  module: kivy.uix.label, cls: None, baseclass: None, filename: None. Ignored -  module: None, cls: None, baseclass: MeasurementItem, filename: /Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/datalogger.kv.     [factory.py:103]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Window: Provider: sdl2     [__init__.py:67]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: Using the "OpenGL ES 2" graphics system     [__init__.py:38]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: Backend used <sdl2>     [__init__.py:44]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: OpenGL version <b'2.1 ATI-4.6.20'>     [__init__.py:48]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: OpenGL vendor <b'ATI Technologies Inc.'>     [__init__.py:49]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: OpenGL renderer <b'AMD Radeon Pro 5500M OpenGL Engine'>     [__init__.py:50]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: OpenGL parsed version: 2, 1     [__init__.py:54]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: Shading version <b'1.20'>     [__init__.py:74]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: Texture max size <16384>     [__init__.py:76]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: Texture max units <16>     [__init__.py:78]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Window: auto add sdl2 input provider     [window_sdl2.py:370]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : Window: virtual keyboard not allowed, single mode, not docked     [__init__.py:2253]
2023-03-31 20:00:14 INFO     [MainThread  ] kivy         : GL: NPOT texture support is available     [__init__.py:271]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 1           - TestSource     [measurementsview.py:36]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 2           - TestSource     [measurementsview.py:36]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 3           - TestSource     [measurementsview.py:36]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.measurem: Adding signal Chuck Temperature > Sensor 4           - TestSource     [measurementsview.py:36]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Trying to restore history from CSV files...     [history.py:178]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195857.csv'     [history.py:188]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195923.csv'     [history.py:188]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195932.csv'     [history.py:188]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195940.csv'     [history.py:188]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Restoring history from CSV file '../logs/csv/signals-2023-03-31-195954.csv'     [history.py:188]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Starting to record history every 1s for 60s     [history.py:70]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy.history : Writing new CSV file '../logs/csv/signals-2023-03-31-200015.csv'     [history.py:130]
2023-03-31 20:00:15 ERROR    [MainThread  ] kivy.__main__: Could not deactivate screensaver     [main.py:74]
2023-03-31 20:00:15 INFO     [MainThread  ] kivy         : Base: Start application main loop     [base.py:520]
2023-03-31 20:00:24 INFO     [MainThread  ] kivy         : Base: Leaving application in progress...     [base.py:616]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/Users/austinibele/VSCodeProjects/hobby_or_dad/data-logger/bin/../main.py", line 90, in <module>     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     DataLoggerApp().run()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/app.py", line 955, in run     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     runTouchApp()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 574, in runTouchApp     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     EventLoop.mainloop()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 339, in mainloop     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     self.idle()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/base.py", line 379, in idle     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     Clock.tick()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 733, in tick     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     self.post_idle(ts, self.idle())     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 689, in idle     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     usleep(1000000 * sleeptime)     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 965, in usleep     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     _usleep(microseconds, self._sleep_obj)     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/lib/python3.9/site-packages/kivy/clock.py", line 556, in _usleep     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     _libc_usleep(int(microseconds))     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr: Exception ignored in: <module 'threading' from '/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py'>     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr: Traceback (most recent call last):     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:   File "/usr/local/Cellar/python@3.9/3.9.16/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 1477, in _shutdown     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr:     lock.acquire()     [logger.py:338]
2023-03-31 20:00:24 WARNING  [MainThread  ] kivy         : stderr: KeyboardInterrupt:      [logger.py:338]
